test/e2e/flowaggregator_test.go

```diff
@@ -343,6 +348,12 @@ func testHelper(t *testing.T, data *TestData, isIPv6 bool) {
 	// perftest-a (Egress reject) -> perftest-b , perftest-a (Egress drop) -> perftest-d
 	t.Run("IntraNodeDenyConnEgressANP", func(t *testing.T) {
 		skipIfAntreaPolicyDisabled(t)
+		label := "IntraNodeDenyConnEgressANP"
+		// As we use the same perftest Pods to generate traffic across all test cases, there's a potential for collecting
```

Comment:
Could we move this comment to our first test case `IntraNodeFlows`?

Reply:
Updated. Thanks!